exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -279,6 +281,17 @@ func (e *exporter) start(_ context.Context, host component.Host) (err error) {
 	return nil
 }
 
+func truncateString(str string, num int) string {
+	truncated := str
+	if len(str) > num {
+		if num > 3 {
+			num -= 3
+		}
+		truncated = str[0:num] + "..."
```

Comment:
Running `truncateString("test", 2)` results in `te...`. This is not _wrong_ but I am wondering if this is what you expect when truncating a string to two chars, and it ends up being longer than 2 chars.

Reply:
Since we only really care about the case of 1000 chars I think we don't need to worry about it. I just wanted to make sure if we reuse this function in the future we don't truncate a string completely away

Reply:
I wanted to include the ellipsis to ensure that a user knows the output was truncated.